package.json

```diff
@@ -22,8 +22,11 @@
     "watch": "concurrent --kill-others 'npm run watch-client' 'npm run localhost'"
   },
   "dependencies": {
-    "react": ">= 0.12.0",
-    "ascii-json": "0.2.0"
+    "ascii-json": "0.2.0",
+    "create-react-class": "15.6",
+    "prop-types": "15.6",
+    "react": "15.6",
+    "react-dom": "15.6"
```

Comment:
Is there a strict requirement on `15.6`, or could this be loosened to `>=15.6` or `~15.6` ([syntax here](https://docs.npmjs.com/files/package.json#dependencies))?

Reply:
Nope, all set to >=15.6 now! There was a vague thought about making sure these moved in lockstep, but that link has me realizing this is what "peer dependencies" are for.